server/container/guice/cassandra-guice/src/test/java/org/apache/james/CassandraJmapTestRule.java

```diff
@@ -47,7 +47,7 @@ public CassandraJmapTestRule(GuiceModuleTestRule... guiceModuleTestRule) {
 
     public GuiceJamesServer jmapServer(Module... additionals) {
         return new GuiceJamesServer()
-            .combineWith(CassandraJamesServerMain.cassandraServerModule)
+            .combineWith(CassandraJamesServerMain.cassandraServerModule, CassandraJamesServerMain.protocols)
```

Comment:
Yes, used by many tests with different protocols.
May be the name of this class is not well chosen: _CassandraJamesTestRule_